lib/components/src/controls/react-editable-json-tree/JsonNodes.tsx

```diff
@@ -0,0 +1,1597 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable react/sort-comp */
+/* eslint-disable react/no-array-index-key */
+
+/* eslint-disable no-undef */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable react/button-has-type */
+
+import React, { cloneElement, Component, ReactElement } from 'react';
```

Comment:
I added all the above files into a single TS files to prevent cyclical dependencies